src/main/java/com/hubspot/jinjava/lib/tag/eager/EagerCallTag.java

```diff
@@ -127,6 +128,7 @@ public String eagerInterpret(
       StringBuilder result = new StringBuilder(
         prefixToPreserveState.toString() + joiner.toString()
       );
+      interpreter.getContext().setDynamicVariableResolver(s -> DeferredValue.instance());
```

Comment:
This may be a dumb question but what does the Call tag do? Looking for it in the docs and struggling to find it

Reply:
It's calls a macro function and then the body of the call tag will be run when within the called macro function `{{ caller() }}` is run.
```
{% macro foo() -%}
1
{{ caller() }}
2
{% endmacro %}
{% call foo() -%}
3
{% endcall %}
```
Would output `132`